spec/lib/swagger_v2_api_declaration.json

```diff
@@ -142,16 +152,6 @@
           "text/xml",
           "text/html"
         ],
-        "parameters": [
```

Comment:
Sorry, thinking about this a little more, just want to talk through it. 

As a user of this gem I'll say it's been useful to have this json its corresponding test file mirror real-life usage, and I think that it's a little confusing to leave the operation-level parameter declarations once the path-level one has been provided.

Also, I feel like the operation-level usage is still tested / illustrated at the `/pets` path block, [here](https://github.com/fotinakis/swagger-blocks/blob/master/spec/lib/swagger_v2_blocks_spec.rb#L59-L77) & [here](https://github.com/fotinakis/swagger-blocks/blob/master/spec/lib/swagger_v2_blocks_spec.rb#L99-L107). 

Totally fine if you still think that re-adding at the operation level is the right call, just wanted to see.